src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableCheckRecords.java

```diff
@@ -0,0 +1,19 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.coding.finallocalvariable;
+
+/* Config:
+ * default
+ */
+public record InputFinalLocalVariableCheckRecords(boolean t, boolean f) {
+    public InputFinalLocalVariableCheckRecords {
+        int a = 0; // ok
+        a = 1;
+    }
+
+    record bad(int i) {
+        public bad {
+            int b = 0;
```

Comment:
The test expects violation at  "15:17", please move `// violation` comment to line 15

Reply:
Fixed. You're right. I thought 17 is the line number. Thank you

Reply:
I'm not sure what went wrong with [regression report](https://github.com/checkstyle/checkstyle/pull/8913#issuecomment-715557963). Do you have any suggestions?